### PR TITLE
Skip the test instead of commenting it out

### PR DESCRIPTION
**Pull request by lubosmj**

### CODE REVIEW

File: tests/databases/test_pymongo.py [hunk 1]
```diff
@@ -140,7 +142,7 @@ def test_database_total_score_eleven(data_single_filter_logical_company_code, da
     assert total_score == 11
 
 
-'''
+@pytest.mark.skip(reason="Cannot mock random sampling")
```
Comment (phanak-sap) [inline tests/databases/test_pymongo.py:145]: The test can be fixed? If yes, how?
If this will never work, then we can delete it instead of skipping.

Comment (lubosmj) [reply tests/databases/test_pymongo.py:145]: Once the issue https://github.com/SAP/odfuzz/issues/9 has been resolved, we will need this test case.

Comment (phanak-sap) [reply tests/databases/test_pymongo.py:145]: Thank you, this is the info I needed.

Comment (phanak-sap) [reply tests/databases/test_pymongo.py:145]: I have added it directly to the code for easier reference than finding out trough necessary search  relevant commit and PR/Issues

**Comment by phanak-sap** [comment]
build is now 
`tests/databases/test_pymongo.py::test_database_sample_filter_entries_no_exclude SKIPPED `

**Comment by phanak-sap** [comment]
Followup issue #50 created.

**Pull request merged**